docs/guides/tdd_bot.md

```diff
@@ -1,24 +1,14 @@
----
```

Comment:
the removal of this is a big deal. have to think about reworking the prompt to respect things like this